LondonBritishLibrary/add/BLadd18993.xml

```diff
@@ -160,6 +160,21 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                                 </item>
                             </list>
                         </additions>
+                        
+                        <bindingDesc>
+                            <binding contemporary="false" xml:id="binding">
+                                <decoNote xml:id="b1" type="Boards">Two boards.</decoNote> 
```

Comment:
What is the material of the boards?

Reply:
I don't know, from what I could see from the images they're completely covered. Could you have a look?

Reply:
Indeed! Difficult.

Reply:
Can it be merged like this, or would you suggest a different formulation?

Reply:
If we do not want to make it complicated, it can be left as it is. If we do, perhaps: "Two boards (of wood or pasteboard)."